### DISCUSSION TITLE
Subpath patterns multiple file extensions

### DISCUSSION THREAD

**Discussion by FrickenHamster** [webpack/webpack#13640] [Q&A]
I'm using webpack 5.40.0
I'm trying to use subpath patterns to sanitize my imports

`"#config/*": "./src/config/*.{js,json}",`
I want to do something like this. I can't get it to work. Is there a way to do this? for now I am just allowing all filetypes.

Thanks

**Comment by alexander-akait** [comment]
What `"#config/*": "./src/config/*.{js,json}",` is mean?

**Reply by FrickenHamster** [reply]
Hey,
Sorry I mean like in the package.json, with the new imports features
For example I have something like this
`  "imports": {
    "#config/*": "./src/config/*.js",
    "#config": "./src/config/index.js",
    "#item/*": "./src/item/*.js",
    "#item": "./src/item/index.js",
  }`
  
This allows me to import config from
`#config` rather than `./config`

the *.js notation allows me to set all js files in that dir as an import, however I want to import both .js and .json files.

**Reply by alexander-akait** [reply]
`{js,json}` is glob and doesn't supported by Node.js/browser/webpack

**Comment by Airkro** [comment]
https://webpack.js.org/api/module-methods/#requirecontext

Alternative:

https://vitejs.dev/guide/features.html#glob-import
https://github.com/OpenSourceRaidGuild/babel-vite
https://github.com/kentcdodds/import-all.macro
https://github.com/snowpackjs/snowpack/pull/2881
